### ISSUE TITLE
Improve highlighted selected block in List View

### ISSUE THREAD

**Issue by talldan** [WordPress/gutenberg#24617]
**Is your feature request related to a problem? Please describe.**
As mentioned on https://github.com/WordPress/gutenberg/pull/24609, it's quite easy to mistake the selected block for the one that has the focus ring in list view:
<img width="267" alt="Screenshot 2020-08-18 at 12 53 27 pm" src="https://user-images.githubusercontent.com/677833/90471895-db88d600-e151-11ea-9d54-8eb6e27225e6.png">

Here, the blue border is the focus ring, and the icon with the black background is the selected block.

**Describe the solution you'd like**
Make the selected block state stronger or more visible. The icon background is very subtle.

**Comment by talldan** [comment]
This may be something being tackled in https://github.com/WordPress/gutenberg/pull/24419. @shaunandrews what do you think ... Can I close this in favor of your issue?

**Comment by talldan** [comment]
Ah yep, I see there are some designs in this comment https://github.com/WordPress/gutenberg/issues/24029#issuecomment-670175528. I'll close this issue in that case.
